.github/workflows/release_charm.yaml

```diff
@@ -3,30 +3,24 @@ name: Upload charm
 on:
   workflow_dispatch:
     inputs:
-      run_id:
+      artifact_id:
         type: number
         required: true
 
 jobs:
   upload_charm:
     runs-on: ubuntu-latest
     steps:
-      - name: download artifacts
-        uses: actions/download-artifact@v4
-        with:
-          repository: canonical/livepatch-server
-          github-token: "${{ secrets.GH_PAT }}"
-          run-id: ${{ inputs.run_id }}
-          merge-multiple: true
-
       - name: load images into local docker registry
         run: |
+          sudo apt install curl
```

Comment:
*Just a suggestion; ignore it anyway to first make it work.*

I'm not sure, but I think `wget` is a better alternative because it's already installed on Ubuntu machines.

Reply:
I also think curl is shipped by default on Ubuntu runners from GitHub. I just added this step to be extra sure, but if it worked, I will switch to wget as you suggested.

Thanks Babak